src/gatsby/src/components/search/filter.js

```diff
@@ -58,10 +58,21 @@ const Filter = ({ filterItems, selectedFilterItems, handleFilterCheck, filterTyp
               checked={checked}
               onChange={event => {
                 handleFilterCheck(item, event)
+                if (typeof window.snowplow === 'function') {
+                  window.snowplow(
+                    "trackStructEvent",
+                    "Checkbox",
+                    event.target.checked ? "Check" : "Uncheck",
+                    `${item.filterType} - ${item.label}`,
```

Comment:
This is good, but you could use the filterType as a property and just pass the label here.